modules/phase_field/src/kernels/MatGradSqCoupled.C

```diff
@@ -0,0 +1,60 @@
+#include "MatGradSqCoupled.h"
+
+template <>
+InputParameters
+validParams<MatGradSqCoupled>()
+{
+  InputParameters params = validParams<Kernel>();
+  params.addCoupledVar("elec", "Electric potential");
```

Comment:
Please use either a verbose name `electric_potential` or a common symbol used for this quantity (`V`).